search/search_integration_tests/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@ set(
   tracer_tests.cpp
 )
 
-omim_add_test(${PROJECT_NAME} ${SRC})
+omim_add_test(
+  ${PROJECT_NAME}
+  LABELS Integration
```

Comment:
Why do we need this label?

Reply:
Just to separate tests in logical scopes and demonstrate what we can do :) I proposed to discuss list of needed labels in this PR.
Anyway routing_integration_tests, for example, will fail without all our *.mwm files in data directory. Labels can help handle of this nuances in code.

Reply:
But search_integration_tests will not fail without all our *.mwm files.

Reply:
Ok, in this specifically case, 'Integration' can be used to divide tests in two parts. 
If unit test 'search_tests' is failed, does it make sense to run 'search_integration_tests'?

Reply:
Surely.

Reply:
Ok, last attempt :) 
Does it make sense to add label 'Search' to all search tests? It can help separate statistic about test passing time for example:
```bash
ctest -L Search

Test project /Users/v.greshilov/MapsMe/omim-temp
    Start 19: search_quality_tests
1/3 Test #19: search_quality_tests .............   Passed    0.04 sec
    Start 20: search_tests
2/3 Test #20: search_tests .....................***Failed    0.76 sec
    Start 21: search_integration_tests
3/3 Test #21: search_integration_tests .........   Passed    9.00 sec

67% tests passed, 1 tests failed out of 3

Label Time Summary:
Search    =   9.80 sec (3 tests)

Total Test time (real) =   9.83 sec
```

Reply:
If you really need this statistics, why not? But, why not to do that for all tests, i.e. why not to add LABEL base for base_tests, LABEL indexer for indexer_tests, etc?

Reply:
BTW, am I right that search_tests are broken?

Reply:
Yes.
As far as I know, World.mwm in the master branch is not renewed yet after #7998

```
Running locality_finder_test.cpp::LocalityFinderTest_Smoke
FAILED
search_tests/locality_finder_test.cpp:73 TEST(result == results[i]) ˈmɛlbn̩ Melbourne
```

Reply:
@ygorshenin I am actually waiting comments from other developers about this PR, and '--amend' it after discuss about labels.
Thanks for comments and attention! Looks like the rest do not care.

Reply:
Integration здесь сбивает с толку, как мне кажется.
Метка "Server" сейчас влияет на строку запуска, например. Если для интеграционных тестов роутинга метка "Integration" будет требовать наличия каких-то mwm в data/, то здесь её быть не должно.

Метку search для поисковых тестов можно и добавить, хотя я и не уверен, что нам нужно такое дублирование.